app/src/main/java/de/danoeh/antennapod/activity/OnlineFeedViewActivity.java

```diff
@@ -154,12 +154,18 @@ protected void onCreate(Bundle savedInstanceState) {
             throw new IllegalArgumentException("Activity must be started with feedurl argument!");
         }
 
-        Log.d(TAG, "Activity was started with url " + feedUrl);
-        setLoadingLayout();
-        if (savedInstanceState == null) {
-            startFeedDownload(feedUrl, null, null);
-        } else {
-            startFeedDownload(feedUrl, savedInstanceState.getString("username"), savedInstanceState.getString("password"));
+        if(feedUrl == null){
+            Toast toast = Toast.makeText(this.getApplicationContext(), "Feed URL not found.", Toast.LENGTH_SHORT);
+            Log.e(TAG, "feedUrl is null.");
+            toast.show();
```

Comment:
You can also use Toast.makeText(...).show directly. Most apps do it like this.

I suggest to include the word "Error" in the toast message.